mosec/coordinator.py

```diff
@@ -37,6 +37,7 @@ def __init__(
         max_batch_size: int,
         stage: str,
         shutdown: Event,
+        shutdown_notif: Event,
```

Comment:
We only need `shutdown` from the py point of view; the notification is for the coordinators to know in advance so that it won't generate the unnecessary error logs.

Reply:
https://www.abbreviations.com/NOTIF
There is this but vscode cannot recognize I think. Yea but we can rename it.
So I thing eventually there should only be two events? One for notification (to suppress error logs) and one for real shtdown.

Reply:
Yes. Two events are enough.